kura/examples/test/org.eclipse.kura.example.wire.bool.multiport.provider.test/src/main/java/org/eclipse/kura/example/wire/bool/multiport/provider/test/BoolLogicTest.java

```diff
@@ -0,0 +1,473 @@
+
+        final BundleContext bundleContext = FrameworkUtil.getBundle(BoolLogicTest.class).getBundleContext();
+
+        builder.addWireComponent(UNDER_TEST_PID, "org.eclipse.kura.wire.LogicalOperators", 2, 1) //
+                .addTestEmitterReceiver(TEST_FIRST_EMITTER_PID) //
+                .addTestEmitterReceiver(TEST_SECOND_EMITTER_PID).addTestEmitterReceiver(TEST_RECEIVER_PID) //
+                .addWire(TEST_FIRST_EMITTER_PID, 0, UNDER_TEST_PID, IN0_PORT) //
+                .addWire(TEST_SECOND_EMITTER_PID, 0, UNDER_TEST_PID, IN1_PORT) //
+                .addWire(UNDER_TEST_PID, OUT_PORT, TEST_RECEIVER_PID, 0);
+
+        try {
+            builder.replaceExistingGraph(bundleContext, wireGraphService).get(30, TimeUnit.SECONDS);
+
+            in1emitter = builder.getTrackedWireComponent(TEST_SECOND_EMITTER_PID);
+            in0emitter = builder.getTrackedWireComponent(TEST_FIRST_EMITTER_PID);
+            outReceiver = builder.getTrackedWireComponent(TEST_RECEIVER_PID);
+        } catch (KuraException | ExecutionException e) {
+            logger.error("Test error", e);
+            throw e;
+        }
+
+        mapWithTrue.put(OPERAND_NAME_DEFAULT, TypedValues.newBooleanValue(true));
+        mapWithFalse.put(OPERAND_NAME_DEFAULT, TypedValues.newBooleanValue(false));
+
+    }
+
+    @Test
+    public void wireComponentExists() throws KuraException {
+        WireGraphConfiguration wgc;
+        try {
+            wgc = wireGraphService.get();
+        } catch (KuraException e) {
+            logger.error("Test error", e);
+            throw e;
+        }
+
+        assertTrue(wgc.getWireComponentConfigurations().stream()
+                .anyMatch(wcc -> UNDER_TEST_PID.equals(wcc.getConfiguration().getPid())));
+    }
+
+    @Test
+    public void wireComponentHasDefaultProperties() throws KuraException {
+        WireGraphConfiguration wgc;
+        try {
+            wgc = wireGraphService.get();
+        } catch (KuraException e) {
+            logger.error("Test error", e);
+            throw e;
+        }
+
+        assertTrue(wgc.getWireComponentConfigurations().stream()
+                .anyMatch(wcc -> matchesDefaultConfiguration(wcc.getConfiguration())));
+    }
+
+    private boolean matchesDefaultConfiguration(ComponentConfiguration cc) {
+        if (cc.getPid().equals(UNDER_TEST_PID)) {
+            Map<String, Object> props = cc.getConfigurationProperties();
+            return OPERAND_NAME_DEFAULT.equals(props.get(FIRST_OPERAND_NAME_PROP_NAME))
+                    && OPERAND_NAME_DEFAULT.equals(props.get(SECOND_OPERAND_NAME_PROP_NAME))
+                    && BARRIER_MODALITY_PROPERTY_DEFAULT == (boolean) props.get(BARRIER_MODALITY_PROPERTY_KEY)
+                    && BOOLEAN_OPERATION_DEFAULT.equals(props.get(BOOLEAN_OPERATION));
+        }
+        return false;
+    }
+
+    @Test
+    public void testAndOperation() throws Exception {
+        logger.info("### TESTING AND OPERATION ###");
```

Comment:
You don't need this log as the test method is always logged during a test execution

Reply:
noted.